src/tooltip/tooltip.js

```diff
@@ -276,6 +284,14 @@ angular.module( 'ui.bootstrap.tooltip', [ 'ui.bootstrap.position', 'ui.bootstrap
               ttScope.title = val;
             });
 
+            function watchNgDisabled() {
+              unbindWatcher = scope.$watch(attrs.ngDisabled, function (newValue, oldValue) {
```

Comment:
I think that would work too. Though I haven't tested what $observe does for boolean attributes like this.